docs/cluster-management/developing-locally.md

```diff
@@ -0,0 +1,184 @@
+    ```bash
+    openssl genrsa -out jwt.pem 2048
+    openssl rsa -in jwt.pem -pubout -out jwt.pub
+    ```
+* Remember to create a folder called "mw-data": `mkdir mw-data`
+
+* Look up your ip first: `ifconfig`, YOUR_IP
+> You may need to update this IP because of location changes.
+
+```
+auth:
+    jwtPrivateKey: |
+        -----BEGIN RSA PRIVATE KEY-----
+        *********SOME KEYS HERE********
+        -----END RSA PRIVATE KEY-----
+
+    jwtPublicKey: |
+        -----BEGIN PUBLIC KEY-----
+        ******SOME KEYS HERE******
+        -----END PUBLIC KEY-----
+
+httpd:
+  # Port to listen on
+  port: 9001
+
+  # Host to listen on (set to localhost to only accept connections from this machine)
+  host: 0.0.0.0
+
+  # Externally routable URI (usually your load balancer or CNAME)
+  # This requires to be a routable IP inside docker for executor, see
+  # https://github.com/screwdriver-cd/screwdriver/blob/095eaf03e053991443abcbde91c62cfe06a28cba/lib/server.js#L141
+  uri: http://YOUR_IP:9001
+
+ecosystem:
+  # Externally routable URL for the User Interface
+  ui: http://sd.screwdriver.cd:4200
+
+  # Externally routable URL for the Artifact Store
+  store: http://YOUR_IP:9002
+
+  allowCors: ['http://sd.screwdriver.cd', 'http://YOUR_IP:9001']
+
+executor:
+    plugin: docker
+    docker:
+        enabled: true
+        options:
+            docker:
+                socketPath: "/var/run/docker.sock"
+
+scms:
+    github:
+        plugin: github
+        config:
+            # github
+            oauthClientId: your-oauth-client-id
+            oauthClientSecret: your-oauth-client-secret
+            secret: a-really-real-secret
+            username: sd-buildbot
+            email: dev-null@screwdriver.cd
+
+datastore:
+  plugin: sequelize
+  sequelize:
+    # Type of server to talk to
+    dialect: sqlite
+    # Storage location for sqlite
+    storage: ./mw-data/storage.db
+```
+
+### store/config/local.yaml
+* Similarly like the `mw-data`, remember to create a folder called "store-data": `mkdir store-data`
+
+```
+auth:
+    # A public key for verifying JWTs signed by api.screwdriver.cd
+    jwtPublicKey: |
+        -----BEGIN PUBLIC KEY-----
+        ******SOME KEYS HERE******
+        -----END PUBLIC KEY-----
+
+strategy:
+    plugin: disk
+    disk:
+        cachePath: './store-data'
+        cleanEvery: 3600000
+        partition : 'cache'
+
+httpd:
+    port: 9002
+
+ecosystem:
+    # Externally routable URL for the User Interface
+    ui: http://sd.screwdriver.cd:4200
+
+    # Externally routable URL for the Artifact Store
+    api: http://sd.screwdriver.cd:9001
+
+    allowCors: ['http://sd.screwdriver.cd']
+```
+
+### Step 5: Install dependencies and you are ready to go!
+Just need to run below commands inside each repo
+
+```
+npm install && npm run start
+```
+
+While all ui, screwdriver and store are runing, now you can go to `http://sd.screwdriver.cd:4200` to check out.
+
+## Developing locally with executor-queue
+
+> TODO
```

Comment:
perhaps @pritamstyz4ever can chime in here ? Haven;t you create a queue based setup for build timeout work ?